tests/Tokenizer/TokensTest.php

```diff
@@ -1375,6 +1375,190 @@ public function provideGetMeaningfulTokenSiblingCases(): \Generator
         yield [null, 0, -1, '<?php /**/ foo();'];
     }
 
+    /**
+     * @dataProvider provideInsertSlicesAtMultiplePlacesCases
+     *
+     * @param array<int, Token> $slices
+     */
+    public function testInsertSlicesAtMultiplePlaces(string $expected, array $slices): void
+    {
+        $input = <<<'EOF'
+<?php
+
+$after = get_class($after);
+$before = get_class($before);
+EOF;
+
+        $tokens = Tokens::fromCode($input);
+        $tokens->insertSlices([
+            16 => $slices,
+            6 => $slices,
+        ]);
+
+        static::assertSame($expected, $tokens->generateCode());
+    }
+
+    public function provideInsertSlicesAtMultiplePlacesCases(): \Generator
+    {
+        yield 'one slice count' => [
+            <<<'EOF'
+<?php
+
+$after =  get_class($after);
+$before =  get_class($before);
```

Comment:
this case is wrong, we must avoid having whitespace next to whitespace as 2 separated tokens!

Reply:
somehow, in line 1398, we check only text-representation of collection. this is wrong, as to whitespace token should not be place one to each other (`new Token([T_WHITESPACE, ' ']), new Token([T_WHITESPACE, ' '])`)
after strong check of `assertTokens`, this case is failing https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/6172/commits/d865593909284817426428716349ee5e27ba1391